palace/linalg/errorestimator.cpp

```diff
@@ -90,14 +96,13 @@ FluxProjector::FluxProjector(const MaterialOperator &mat_op,
                              bool pa_discrete_interp)
 {
   BlockTimer bt(Timer::CONSTRUCTESTIMATOR);
-  constexpr int skip_zeros = 0;
+  constexpr int skip_zeros = false;
```

Comment:
Looking at `ParOperator::Assemble`, these are bools, so probably worth just changing the type and being done with it.

Reply:
Whoops, I only caught half the issue. Thanks!

Reply:
Fixed by c581aae.